doc/tutorials/using_partition_mode_on_nuc.rst

```diff
@@ -150,7 +148,7 @@ Update ACRN Hypervisor Image
 
       $ git clone https://github.com/projectacrn/acrn-hypervisor.git
       $ cd acrn-hypervisor
-      $ git checkout v2.4
+      $ git checkout v2.6
 
 #. Check the ``pci_devs`` sections in ``misc/config_tools/data/whl-ipc-i7/logical_partition.xml``
```

Comment:
That board (`whl-ipc-i7`) is being removed by #6590 . We need to use a new board for this tutorial